jax/_src/lax/linalg.py

```diff
@@ -1441,3 +1441,120 @@ def tridiagonal_solve(dl, d, du, b):
     raise ValueError(f'Only f32/f64 are supported, got {t}')
 
   return tridiagonal_solve_p.bind(dl, d, du, b, m=m, n=n, ldb=ldb, t=t)
+
+
+# Schur Decomposition
+
+
+def schur(x,
+          compute_schur_vectors=True,
+          sort_eig_vals=False,
+          select_callable=None):
+  return schur_p.bind(
+      x,
+      compute_schur_vectors=compute_schur_vectors,
+      sort_eig_vals=sort_eig_vals,
+      select_callable=select_callable)
+
+
+def schur_impl(operand, *, compute_schur_vectors, sort_eig_vals,
```

Comment:
Slight preference for all of these implementation details to be marked private with a leading underscore, e.g.
```
def _schur_impl(...)
```
Same below.

Reply:
Thank you for reviewing ! I have marked them private